### DISCUSSION TITLE
Baba ghanoush is typically made from which vegetable?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#429] [Galaxy Trivia] [answered]
**Bonus:** If you don't have it in your kitchen, what is the second most popular vegetable to substitute?

**Comment by peterbaccaro** [comment, marked as answer]
Baba ghanoush is typically made with **aubergines** (also known as **eggplant**).
A popular substitute for the eggplant would be **courgettes** (also known as **zucchini**).

**Reply by ParanoidUser** [reply]
Woo-hoo! You made it up to the gold! 👏👏 Congratulations on your well deserved [x4 Galaxy Brain 🥇](https://github.com/peterbaccaro?tab=achievements&achievement=galaxy-brain) badge. Don't stop, there are more interesting questions and challenging puzzles to come. As always don't hesitate to post your own questions to the community. 😊🎉

**Comment by thuongtruong1009** [comment]
Baba ghanoush is a popular Middle Eastern dip that is typically made from eggplant (aubergine). The eggplant is roasted or grilled until it is soft and tender, then the flesh is scooped out and mashed with other ingredients such as tahini, lemon juice, garlic, and olive oil. The resulting dip has a creamy texture and a smoky, slightly sweet flavor that is often served as a mezze (appetizer) or side dish. Baba ghanoush is a vegetarian and vegan dish and is a good source of fiber, vitamins, and minerals.
